README.md

```diff
@@ -355,50 +355,67 @@ If you open the `build.ps1` script at the beginning you will find all available
 
 To run Darknet on Linux use examples from this article, just use `./darknet` instead of `darknet.exe`, i.e. use this command: `./darknet detector test ./cfg/coco.data ./cfg/yolov4.cfg ./yolov4.weights`
 
 ### How to compile on Windows (using `CMake`)
 
 Requires:
 
-* MSVC: https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=Community
-* CMake GUI: `Windows win64-x64 Installer`https://cmake.org/download/
-* Download Darknet zip-archive with the latest commit and uncompress it: [master.zip](https://github.com/AlexeyAB/darknet/archive/master.zip)
+- MSVC: https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=Community
+- CMake GUI: `Windows win64-x64 Installer`https://cmake.org/download/
+- Download Darknet zip-archive with the latest commit and uncompress it: [master.zip](https://github.com/AlexeyAB/darknet/archive/master.zip)
 
 In Windows:
 
-* Start (button) -> All programs -> CMake -> CMake (gui) ->
+- Start (button) -> All programs -> CMake -> CMake (gui) ->
 
-* [look at image](https://habrastorage.org/webt/pz/s1/uu/pzs1uu4heb7vflfcjqn-lxy-aqu.jpeg) In CMake: Enter input path to the darknet Source, and output path to the Binaries -> Configure (button) -> Optional platform for generator: `x64`  -> Finish -> Generate -> Open Project ->
+- [look at image](https://habrastorage.org/webt/pz/s1/uu/pzs1uu4heb7vflfcjqn-lxy-aqu.jpeg) In CMake: Enter input path to the darknet Source, and output path to the Binaries -> Configure (button) -> Optional platform for generator: `x64`  -> Finish -> Generate -> Open Project ->
 
-* in MS Visual Studio: Select: x64 and Release -> Build -> Build solution
+- in MS Visual Studio: Select: x64 and Release -> Build -> Build solution
 
-* find the executable file `darknet.exe` in the output path to the binaries you specified
+- find the executable file `darknet.exe` in the output path to the binaries you specified
 
 ![x64 and Release](https://habrastorage.org/webt/ay/ty/f-/aytyf-8bufe7q-16yoecommlwys.jpeg)
 
-
 ### How to compile on Windows (using `vcpkg`)
 
 This is the recommended approach to build Darknet on Windows.
 
-1. Install Visual Studio 2017 or 2019. In case you need to download it, please go here: [Visual Studio Community](http://visualstudio.com). Remember to install English language pack, this is mandatory for vcpkg!
+1. Open Powershell as Administrator (Start -> All programs -> right click on Windows Powershell and choose "Run as administrator") and type this command:
 
-2. Install CUDA (at least v10.0) enabling VS Integration during installation.
+```PowerShell
+PS />              Set-ExecutionPolicy unrestricted
```

Comment:
@AlexeyAB  this line should help users

Reply:
sorry for the noise in the README. The linter cleaned up a lot of mismatches and I didn't want to stop it 🤣